easy_access/services/merge.py

```
@@ -0,0 +1,301 @@
+        setattr(db_item, field, new_value)
+    return changes
+
+
+def _cast_datetime_value(value: Any) -> datetime | None:
+    """Cast a value to datetime with multiple format fallbacks."""
+    if not value:
+        return None
+
+    try:
+        return datetime.strptime(value, "%Y-%m-%d %H:%M:%S%z").replace(tzinfo=UTC)
+    except Exception:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)